### ISSUE TITLE
error when running shell script file in directory examples/03_checkerboard_2d

### ISSUE THREAD

**Issue by ilovesons** [xumi1993/SurfATT-iso#31]
After successfully installing surfatt, I tried to run examples provided in github page.
Examples '00_checkerboard_iso' and '04_hawaii_topo' are executed without any problem,
but when I try to run shell script file in example '03_checkerboard_2d',
I get following messages.

```
 /output does not contain key "verbose_level".
 Error detected, aborting MPI... proc            0
--------------------------------------------------------------------------
MPI_ABORT was invoked on rank 0 in communicator MPI_COMM_WORLD
  Proc: [[38992,1],0]
  Errorcode: 30

NOTE: invoking MPI_ABORT causes Open MPI to kill all MPI processes.
You may or may not see output from other processes, depending on
exactly when Open MPI kills them.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
prterun has exited due to process rank 0 with PID 0 on node pago calling
"abort". This may have caused other processes in the application to be
terminated by signals sent by prterun (as reported here).
--------------------------------------------------------------------------
```

It would be greatly appreciated if you give me a suggestion for solving this problem.
Thanks.

**Comment by xumi1993** [comment]
#This example includes the `input_params.yml` with an old version. I will update this file ASAP.

**Comment by xumi1993** [comment]
This issue has been fixed by PR #32

**Comment by ilovesons** [comment]
Thank you so much for your fast feedback!
Now everything works well.
